shared/app/main.native.js

```diff
@@ -36,13 +36,9 @@ class Main extends React.Component<Props> {
   }
 
   render() {
-    if (this.props.showPushPrompt) {
```

Comment:
if we don't do this then when we show the push prompt it thrashes the entire stack. for nav2 this means the ref to the navigator gets lost so we can lose sending actions to it!